### DISCUSSION TITLE
BT-04-Notice

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#572] [Q&A] [answered]
Hi Lab !
a simple question, I think
Which notice share the same Procedure Identifier ? ( CFC notice, CN and CAN ? ) Or every notice except for  [ "1", "2", "3", "4", "5", "6", "7", "8", "9", "T01", "X01", "X02" ] , has got his own ?
Thanks and Regards
Renato

**Comment by YvesJo** [comment, marked as answer]
Hi Renato,
For questions non related to the SDK, please contact the Helpdesk at [ted@publications.europa.eu](mailto:ted@publications.europa.eu).
All the notices belonging to a same procurement procedure, shall share the same procedure ID. This does not apply to Planning notices (1 - 9, T01) and to notices not related to Procurement Procedures (X01, X02) for which there is no defined BT-04.
